tests/bin/non-github-doc-uri/run.t

```diff
@@ -95,9 +95,7 @@ We do the whole dune-release process
     [ OK ] package(s) pass the tests
     
     [+] Distribution for whatever 0.1.0
-    [+] Commit ...
     [+] Archive _build/whatever-0.1.0.tbz
-    [1]
```

Comment:
Is it intentional to remove this line? Those tests need serious rework but until then, I think the exit status actually is valuable here!

Reply:
You're right, I have to investigate why the exit code changes suddenly. That's most likely wrong, given the package has linting errors.

Reply:
So after I looked into this it is because now the error state reported is of `grep` and not `dune-release`. I've committed a fix but it only works on systems that use Bash as `$PIPESTATUS` is a bashism. Checking CI that means it works on Fedora, CentOS, OpenSUSE but fails on Debian, Ubuntu and Alpine. I'm not particularly fond of this solution so maybe we should just not test the error code in this case.

Reply:
I think the exit status ins't that important here and checking it prints the error message we're after should be enough.
This test could really use a description btw, it's really hard to understand what we're trying to test.

I assume we want to see the error messages for github URLs parsing failure, I guess all the rest is pretty much worthless.

Could you have a look at the original PR see what this is supposed to test so we can polish it a bit (in a separate PR)?

Reply:
I think the original PR is #256 and I think you're right, the only thing that is important is the error message. I would suggest at some point going through all the tests, identifying what is important and then grepping for that exact thing.